src/jit/gentree.h

```diff
@@ -2547,7 +2552,7 @@ struct GenTreeCall final : public GenTree
     // TODO-AllArch: enable for all call nodes to unify single-reg and multi-reg returns.
     ReturnTypeDesc*   GetReturnTypeDesc()
     {
-#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
+#if FEATURE_MULTIREG_RET
```

Comment:
TODO comments lines 2550 and 2551 can be deleted now.
Also the NOTE: can be deleted now.